### PR TITLE
Adding Help & Contribute page [#101]

### PR DESCRIPTION
**Pull request by bmorris3**
For #101, partly borrowed from `photutils`. cc @cdeil

### CODE REVIEW

File: docs/faq/contribute.rst [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+* `Comprehensive Coding Guidelines <http://docs.astropy.org/en/latest/development/codeguide.html>`_
+
+* `Astropy Developer Documentation <http://docs.astropy.org/en/latest/#developer-documentation>`_
+
+.. note::
+   `astroplan` is a very young project and we welcome new features. If you are
+   interested in contributing, contact us via
+   `GitHub <https://github.com/astropy/astroplan>`_ or the
+   `astropy dev mailing list <https://groups.google.com/forum/#!forum/astropy-dev>`_
+   to check what's already in progress and what
```
Comment (cdeil) [inline docs/faq/contribute.rst]: This sentence is cut off.
Otherwise :+1: on location and wording of this info!

File: docs/faq/contribute.rst [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+***************************
+Getting Help & Contributing
+***************************
+
+Getting Help
+============
+
+If your questions are not addressed in the docs, you can reach out to us for
+help at the
+`astropy dev mailing list <https://groups.google.com/forum/#!forum/astropy-dev>`_.
```
Comment (eteq) [inline docs/faq/contribute.rst]: ``astropy dev`` -> ``astropy-dev``?

File: docs/faq/contribute.rst [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+* `Comprehensive Coding Guidelines <http://docs.astropy.org/en/latest/development/codeguide.html>`_
+
+* `Astropy Developer Documentation <http://docs.astropy.org/en/latest/#developer-documentation>`_
+
+.. note::
+   `astroplan` is a very young project and we welcome new features. If you are
+   interested in contributing, contact us via
+   `GitHub <https://github.com/astropy/astroplan>`_ or the
+   `astropy dev mailing list <https://groups.google.com/forum/#!forum/astropy-dev>`_
```
Comment (eteq) [inline docs/faq/contribute.rst]: (ditto)

**Comment by eteq** [comment]
super-minor comments above, but looks good otherwise

**Comment by bmorris3** [comment]
Great. Corrected and merging!

**Pull request merged**